pyproject.toml

```diff
@@ -0,0 +1,2 @@
+[build-system]
+requires = ["setuptools>=50.0", "wheel"]
```

Comment:
If you are going to use pyproject.toml, maybe think about migrating stuff from setup.py to pyproject.toml...

Reply:
Attempted!